src/mx_bluesky/hyperion/experiment_plans/flyscan_xray_centre_plan.py

```diff
@@ -286,14 +286,18 @@ def _xrc_result_in_boxes_to_result_in_mm(
     xray_centre = fgs_params.grid_position_to_motor_position(
         np.array(xrc_result["centre_of_mass"])
     )
+    # A correction is applied to the bounding box to map discrete grid coordinates to
+    # the corners of the box in motor-space; we do not apply this correction
+    # to the xray-centre as it is already in continuous space and the conversion has
+    # been performed already
     return XRayCentreResult(
         centre_of_mass_mm=xray_centre,
         bounding_box_mm=(
```

Comment:
As discussed the entire logic here is quite hard to explain via comments, but I think this helps to explain the 0.5
```suggestion
        # Bounding box coordinates give position of the centre of the box, subtract half a box size to get the start of the box
        bounding_box_mm=(
```

Reply:
Whoops, didn't mean to delete line 294 in this suggestion